src/patternfly/components/AppLauncher/app-launcher.scss

```
@@ -73,7 +73,7 @@
   --pf-c-app-launcher__group-title--PaddingBottom: var(--pf-c-app-launcher__menu-item--PaddingBottom);
   --pf-c-app-launcher__group-title--PaddingLeft: var(--pf-c-app-launcher__menu-item--PaddingLeft);
   --pf-c-app-launcher__group-title--FontSize: var(--pf-global--FontSize--sm);
```

should this be `--xs`?

```suggestion
  --pf-c-app-launcher__group-title--FontSize: var(--pf-global--FontSize--xs);
```

---

OK will update and apply across others noted below